src/array.rs

```diff
@@ -21,6 +21,21 @@ impl<T: RuntimeType> Array<T> {
         Self::default()
     }
 
+    pub fn with_len(len: usize) -> Self {
+        unsafe {
+            assert!(len < 0xFFFF_FFFF as usize);
```

Comment:
You can use the constant instead: https://doc.rust-lang.org/stable/std/u32/constant.MAX.html this has been there since 1.0